src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -327,7 +327,6 @@ public override void DefaultVisit(Symbol symbol)
                     symbol,
                     shouldSkipPartialDefinitionComments,
                     docCommentNodes,
-                    reportParameterOrTypeParameterDiagnostics,
```

Comment:
Consider moving the local variable down near its actual usage to avoid calling `GetLocationInTreeReportingDocumentationCommentDiagnostics` unnecessarily in cases where we return.